docs/rampviz/plugins.rst

```diff
@@ -62,6 +62,17 @@ Slice
 Ramp Extraction
 ===============
 
-Coming soon.
-
-
+Extract a ramp from a ramp cube.
+
+Data products from infrared detectors flow through the official
+`JWST <https://jwst-pipeline.readthedocs.io/en/latest/>`_ or
+`Roman <https://roman-pipeline.readthedocs.io/en/latest/>`_ mission pipelines in levels. Infrared detectors use
```

Comment:
Ideally this should also be using `:ref:` but I cannot find a good place to do it where it exist already. Might need to add one upstream or maybe use `:doc:` as you have.

Reply:
xref https://github.com/spacetelescope/romancal/pull/1529